OrbitGl/ProcessManager.cpp

```diff
@@ -14,7 +14,8 @@
 
 namespace {
 
-constexpr uint64_t kGrpcCallTimeoutMilliseconds = 1000;
+constexpr uint64_t kGrpcDefaultTimeoutMilliseconds = 1000;
+constexpr uint64_t kGrpcSymbolLoadingTimeoutMilliseconds = 10000;
```

Comment:
Is this enough?

Reply:
It should be, but I gave it a very generous 60seconds and wrote a comment